lib/sequent/core/aggregate_root.rb

```diff
@@ -40,6 +40,8 @@ class AggregateRoot
       include Helpers::AutosetAttributes
       include SnapshotConfiguration
 
+      message_base_class Sequent::Core::Event
```

Comment:
I would prefer to not configure this here, but to keep in the MessageHandler as a Hash or something. Reason is that it is quite out of context here imo.

Reply:
Yes, we only have `EventHandler` and `CommandHandler`. I'm going to give it another try with 'runtime' checking. We probably won't need this then.